### ISSUE TITLE
Any progress on One UI 5.1 

### ISSUE THREAD

**Issue by Genxster1998** [BLuFeNiX/SamsungTTSSystemShell#1]
Do you know what was patched ? , It's package manager or selinux context ?
Someone has bypassed the patch , any idea on progress?

**Comment by SteadyStatus21** [comment]
What was patched was the downgrading of the Samsung SMT app, which Google / Android put a stop to. They prevented you from downgrading further than what came with the update on your phone. Not sure what is going to happen here.

**Comment by wr3cckl3ss** [comment]
Progress was made up until Google pulled the plug. But to bypass the One UI 5.1 patch, now that's still in our favor. if you know where to look, then it's easily available. But I do have an option coming that updates and upgrades SMT Shell with a twist. I will upload very very soon and you and others can judge for yourself.

**Comment by Lumince** [comment]
So as of the latest June/July patch this may never be possible again? I updated a few days ago sadly...

**Comment by SteadyStatus21** [comment]
> So as of the latest June/July patch this may never be possible again? I updated a few days ago sadly...

Not necessarily, as wr3cckl3ss1 said, someone has to discover where to look to find the bypass. Time will tell if someone discovers it and reports it back to Samsung first or the public first.

**Comment by Lumince** [comment]
> > So as of the latest June/July patch this may never be possible again? I updated a few days ago sadly...
> 
> Not necessarily, as wr3cckl3ss1 said, someone has to discover where to look to find the bypass. Time will tell if someone discovers it and reports it back to Samsung first or the public first.

Do you know what the last supported update was for the S23 Ultra by chance? 
![Screenshot_20230718_100842_Settings](https://github.com/BLuFeNiX/SMTShell/assets/10453497/ca6841f3-b63a-418d-b282-a2bf1ca2ae99)
I have S1 as the bootloader and I'm seeing a March 2023 update that I might be able to downgrade too?
https://samfrew.com/download/Galaxy__S23__Ultra__/uyhc/USC/S918U1UES1AWC9/S918U1OYM1AWC9/

**Comment by wr3cckl3ss** [comment]
Don't hold out thinking that a system shell is not possible anymore....SMT, on the other hand, is 100 percent over and done with. 

S23 Ultra July Firmware

![Screenshot_20230716_162335_FTL.jpg](https://github.com/BLuFeNiX/SMTShell/assets/97212092/b20580bb-1771-4787-9950-5473d1242235)


It would be interesting, if @BLuFeNiX got in the mix 😉😉😉

**Comment by Lumince** [comment]
> Don't hold out thinking that a system shell is not possible anymore....SMT, on the other hand, is 100 percent over and done with.
> 
> S23 Ultra July Firmware
> 
> ![Screenshot_20230716_162335_FTL.jpg](https://user-images.githubusercontent.com/97212092/254297382-b20580bb-1771-4787-9950-5473d1242235.jpg)
> 
> It would be interesting, if @BLuFeNiX got in the mix 😉😉😉

Woah! Is that your stuff working on the July update?! OMG it is!

**Comment by wr3cckl3ss** [comment]
Technically speaking, none of it ...is mine, as it is a system shell, it's all theirs. The only thing here I can call mine was the research, effort and middle of the night Mad Hatter experiments. Currently there is no release date and no timetable to release any more info on it, as even I...am shocked at how this works. So before anyone enjoys this, I think I'll play around a bit first. See ya soon.

**Comment by Lumince** [comment]
> Technically speaking, none of it ...is mine, as it is a system shell, it's all theirs. The only thing here I can call mine was the research, effort and middle of the night Mad Hatter experiments. Currently there is no release date and no timetable to release any more info on it, as even I...am shocked at how this works. So before anyone enjoys this, I think I'll play around a bit first. See ya soon.

I look forward to seeing what comes out of it! Do you have a patreon or anything that I could donate too?

**Comment by wr3cckl3ss** [comment]
> > Technically speaking, none of it ...is mine, as it is a system shell, it's all theirs. The only thing here I can call mine was the research, effort and middle of the night Mad Hatter experiments. Currently there is no release date and no timetable to release any more info on it, as even I...am shocked at how this works. So before anyone enjoys this, I think I'll play around a bit first. See ya soon.
> 
> I look forward to seeing what comes out of it! Do you have a patreon or anything that I could donate too?

I'm flattered you would want to donate to me but my intentions here were never to gain  financially but to gain knowledge...maybe one day in future, I'll revisit the idea of maybe taking a donation to something I contributed greatly to the community but i still have a lot to learn and to date, since I started, I've grasped quite a bit. So that's paying off....in itself. I believe I have my Twitter and Reddit links on my profile, you're more than welcome to follow me there. I do have other platforms I use as well but I tend to NOT give those out anymore because I get flooded with messages. Stay tuned...

**Comment by Lumince** [comment]
> > > Technically speaking, none of it ...is mine, as it is a system shell, it's all theirs. The only thing here I can call mine was the research, effort and middle of the night Mad Hatter experiments. Currently there is no release date and no timetable to release any more info on it, as even I...am shocked at how this works. So before anyone enjoys this, I think I'll play around a bit first. See ya soon.
> > 
> > 
> > I look forward to seeing what comes out of it! Do you have a patreon or anything that I could donate too?
> 
> I'm flattered you would want to donate to me but my intentions here were never to gain financially but to gain knowledge...maybe one day in future, I'll revisit the idea of maybe taking a donation to something I contributed greatly to the community but i still have a lot to learn and to date, since I started, I've grasped quite a bit. So that's paying off....in itself. I believe I have my Twitter and Reddit links on my profile, you're more than welcome to follow me there. I do have other platforms I use as well but I tend to NOT give those out anymore because I get flooded with messages. Stay tuned...

Intentional or not, you have provided the Samsung community with something amazing that I haveny seen in the US for a long time. Actual control over our devices. I initially found your projects after trying to disable 5G on my S23 Ultra and seeing that Samsung completely disabled my option to do so. You helped my partner and family be able to disable 5G on their locked down devices!!

I sadly no longer use those 2 platforms! I avoid 90% of social media haha The only thing I have is Discord. I followed you here and plan to just hold out and block updates entirely so I can wait for the magic that I see that you have posted! Thank you for all of the hard work!

**Comment by ItsZer0here** [comment]
I'm also interested in this shell but I have one UI 5.1.1, which apparently made changes to 5.1, probably rubbing more salt in this wound
